src/AccessTokenController.php

```diff
@@ -5,23 +5,22 @@
 
 
 use Exception;
-use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Config;
 use Illuminate\Support\Facades\Redirect;
 
 class AccessTokenController
 {
-    public function handleRedirect(Request $request)
+    public function handleRedirect()
```

Comment:
We can still inject the $request into the method as you had it before, then we get rid of the helper method, seeing as all the others are gone. The test will still pass (as long as there is no type-hint on the requestToken method in the Profile class)

Reply:
This is correct because `\Illuminate\Foundation\Testing\Concerns\MakesHttpRequests::get` is able to inject the request